angular-spa-ui/js/search-app/tags-module/tags-ctrl.js

```diff
@@ -1 +1,81 @@
 "use strict";
+
+//var tagsCtrl = {};
+
+module.exports = function($scope, $http) {
```

Comment:
When this module initializes it send 7 _GET requests to the server. Why you did it?